package.json

```diff
@@ -95,6 +95,7 @@
     ],
     "transform": {
       "^.+\\.js$": "babel-jest"
-    }
+    },
+    "testURL": "http://localhost"
```

Comment:
Note: needed to add this for tests to pass (even before changes). See https://github.com/facebook/jest/issues/6766.